metricflow/aggregation_properties.py

```diff
@@ -15,7 +15,7 @@ def is_expansive(agg_type: AggregationType) -> bool:
         AggregationType.SUM,
         AggregationType.MIN,
         AggregationType.MAX,
-        AggregationType.BOOLEAN,
+        AggregationType.SUM_BOOLEAN,
```

Comment:
@QMalcolm Also checking that boolean got removed.

Reply:
Yup, same PR. https://github.com/dbt-labs/dbt-semantic-interfaces/pull/33/commits/44ea7fba8b6e67f45ed5b8c599109697f872fd23#diff-d7f77c3d98a0690d5778a5a035d0214589902735434bed98c9d13b992609ecc7L11-L12